file_entity.tokens.inc

```diff
@@ -95,10 +97,11 @@ function file_entity_tokens($type, $tokens, array $data = array(), array $option
     // Chained token relationships.
     $token_service = \Drupal::service('token');
     if (($file_type_tokens = $token_service->findWithPrefix($tokens, 'type')) && $file_type = FileType::load($file->bundle())) {
-      $replacements += $token_service->generate('file-type', $file_type_tokens, array('file_type' => $file_type), $options);
+      $bubbleable_metadata->addCacheableDependency($file_type);
```

Comment:
This line is not necessary because it's an object we pass to generate. Everything that's passed in our out as part of $data is automatically tracked. The first call is correct!